src/YoloDev.Expecto.TestSdk/settings.fs

```diff
@@ -14,57 +19,155 @@ type RunSettings =
     disableParallelization: bool
 
     /// Gets a value which indicates the target framework the tests are being run in.
-    targetFrameworkVersion: string option }
+    targetFrameworkVersion: string option 
+
+    /// Gets the [ExpectoConfig](https://github.com/haf/expecto#the-config) that was set via RunSettings
+    expectoConfig : ExpectoConfig }
 
 [<RequireQualifiedAccess>]
 module RunSettings = 
   let defaultSettings = 
     { collectSourceInformation = true
       designMode = true
       disableParallelization = false
-      targetFrameworkVersion = None }
-  
-  let read (runSettings: IRunSettings) = 
-    let settings = defaultSettings
+      targetFrameworkVersion = None
+      expectoConfig = ExpectoConfig.defaultConfig }
+
+  let private foldCLIArgumentToConfig = function
```

Comment:
Is this function copied from Expecto? If it is, could you add a comment which links to it? Also, if there isn't an issue available on making this public, we should probably make one.